static/js/vue-cdr-access/tests/unit/aggregateRecord.spec.js

```diff
@@ -0,0 +1,735 @@
+}
+
+let wrapper, router;
+
+describe('aggregateRecord.vue', () => {
+    const i18n = createI18n({
+        locale: 'en',
+        fallbackLocale: 'en',
+        messages: translations
+    });
+
+    beforeEach(() => {
+        moxios.install();
+        router = createRouter({
+            history: createWebHistory(process.env.BASE_URL),
+            routes: [
+                {
+                    path: '/record/:uuid',
+                    name: 'displayRecords',
+                    component: displayWrapper
+                }
+            ]
+        });
+
+        wrapper = shallowMount(aggregateRecord, {
+            global: {
+                plugins: [i18n, router]
+            },
+            props: {
+                recordData: record
+            }
+        });
+    });
+
+    afterEach(() => {
+        wrapper = null;
+        router = null
+    });
+
+    it("displays breadcrumbs", () => {
+        expect(wrapper.findComponent({ name: 'breadCrumbs' }).exists()).toBe(true);
+    });
+
+    it("displays a work title", () => {
+        expect(wrapper.find('h2').text()).toEqual('Listen for real');
+    });
+
+    it("does not display a finding aid link, if absent", () => {
+        expect(wrapper.find('.finding-aid').exists()).toBe(false);
+    });
+
+    it("displays a finding aid link, if present", async () => {
+        let updated_record = cloneDeep(record);
+        updated_record.findingAidUrl = 'https://unc-finding-aid.lib.unc.edu';
+        await wrapper.setProps({
+            recordData: updated_record
+        });
+        expect(wrapper.find('.finding-aid').exists()).toBe(true);
+    });
+
+    it("does not allow users to edit the work by default", async () => {
+        let updated_record = cloneDeep(record);
+        updated_record.briefObject.permissions = [];
+        await wrapper.setProps({
+            recordData: updated_record
+        });
+        expect(wrapper.find('a.edit').exists()).toBe(false);
+    });
+
+    it("allows users to edit the work with the proper permissions", async () => {
+        expect(wrapper.find('a.edit').exists()).toBe(true);
+    });
+
+    it("does not set a download link by default", async () => {
+        let updated_record = cloneDeep(record);
+        updated_record.briefObject.permissions = [];
+        updated_record.briefObject.groupRoleMap = {
+            everyone: 'canViewMetadata',
+            authenticated: 'canViewAccessCopies'
+        }
+        await wrapper.setProps({
+            recordData: updated_record
+        });
+        expect(wrapper.find('a.download').exists()).toBe(false);
+    });
+
+    it("allows users to download the work if a download link is present", async () => {
+        let updated_record = cloneDeep(record);
+        updated_record.briefObject.dataFileUrl = 'https://download-link.lib.unc.edu';
+        await wrapper.setProps({
+            recordData: updated_record
+        });
+        expect(wrapper.find('a.download').exists()).toBe(true);
```

Comment:
it'd be good to verify that the download link goes to the correct url with a pid and stuff.